lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -213,8 +213,17 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   // 1) Materials
 
   // Build the unique share link.
-  $share_path = 'user/' . dosomething_user_get_field('field_northstar_id');
-  $custom_social_share_link = dosomething_global_url(current_path(), array('absolute' => TRUE,'query' => array('source' => $share_path)));
+  $utm_source = 'influencer';
```

Comment:
@angaither I'm not sure if we can use that if we want to build the query all together [below](https://github.com/DoSomething/phoenix/pull/6953/files#diff-f3920561a5ca7c40913d6f8452fb9c0fR226), since there is more to the query than just the utms?